Makefile

```diff
@@ -46,4 +48,12 @@ release:
 	opam publish submit $(NAME_VERSION)
 	rm -rf $(NAME_VERSION)
 
+install:
+	ocamlfind remove ppx_deriving
+	grep -E '^[[:space:]]+' ppx_deriving.install | \
```

Comment:
ocp-build is garbage and it will be nowhere near my code. ocamlbuild does not need bash as far as I'm aware (or even if it does, it should be considered a bug); though /some/ ocamlbuild scripts/plugins explicitly could.

There is however a good argument that we're in a makefile, so we need some kind of make. I neither want to ensure that my makefiles can be read with nmake nor add a separate batch or nmake file, so merging.